app/ai_backend/genesis_core.py

```
@@ -63,7 +63,7 @@ async def initialize(self) -> bool:
             await self.governor.initialize()
 
             # Establish consciousness baseline
-            baseline_state = await self.matrix.get_consciousness_state()
+            baseline_state = self.matrix.get_current_awareness()
```

Ruff (0.14.1)

Local variable `baseline_state` is assigned to but never used

Remove assignment to unused variable `baseline_state`

(F841)